ibm-cloud-pak-for-data/additionalnodes.json

```diff
@@ -0,0 +1,864 @@
+{
```

Comment:
it's a little hard to reason over this to figure out how it's different than azuredeploy.json...  why is it split out into it's own template?  The recommended practice would be to use the same template with the proper options...  What's different in this case?